pkg/resources/scim_integration_acceptance_test.go

```diff
@@ -1,79 +1,180 @@
 package resources_test
 
 import (
-	"fmt"
 	"testing"
 
 	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/helpers"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/snowflakeroles"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
 
+	"github.com/hashicorp/terraform-plugin-testing/config"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
-func TestAcc_ScimIntegration(t *testing.T) {
-	scimIntName := acc.TestClient().Ids.Alpha()
-	scimProvisionerRole := "AAD_PROVISIONER"
-	scimNetworkPolicy := acc.TestClient().Ids.Alpha()
+func TestAcc_ScimIntegration_basic(t *testing.T) {
+	networkPolicy, networkPolicyCleanup := acc.TestClient().NetworkPolicy.CreateNetworkPolicy(t)
+	t.Cleanup(networkPolicyCleanup)
 
+	role, roleCleanup := acc.TestClient().Role.CreateRoleWithRequest(t, sdk.NewCreateRoleRequest(snowflakeroles.GenericScimProvisioner).WithOrReplace(true))
```

Comment:
Hmm 🤔 , I thought these are predefined like SECURITYADMIN, ACCOUNTADMIN, etc.

Reply:
They aren't, that's why they need to be provided here.